docfiles/docs.js

```diff
@@ -47,8 +54,21 @@ function searchSubmit(form) {
 }
 
 function setupSidebar() {
+    let togglesidebar = document.getElementById("togglesidebar");
```

Comment:
It is supported from IE 11 on Windows 8.1. In my country (at least) it is true that a lot of schools are still on Windows 7.